### PR TITLE
[ch20439] Add cd pipeline so we can have a stable latest release

### PR DESCRIPTION
**Pull request by victorpoluceno**
Our release process was not working well, as the dev changes would break the stable chart version. These PR adds a CI and CD pipeline. The CI will deploy Docker images tagged with branch name and commit hash. The chart version will be 0.0.1 plus commit hash. The CD pipeline will deploy the latest Docker image and the chart version pinned in the chart.yaml.

### CODE REVIEW

File: .codefresh/auto-prune/pipeline.yml [hunk 1]
```diff
@@ -8,6 +8,6 @@ steps:
     type: freestyle
     stage: run
     # This has to be the full image uri otherwise it doesn't work
-    image: ${{IMAGE_REGISTRY_URL}}/${{IMAGE_NAME}}
+    image: ${{IMAGE_REGISTRY_URL}}/${{IMAGE_NAME}}:latest
```
Comment (victorpoluceno) [inline .codefresh/auto-prune/pipeline.yml:11]: We should always use latest.

File: .codefresh/ci/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    type: helm
+    working_directory: ./cf-review-env
+    arguments:
+      action: push
+      helm_version: ${{K8S_HELM_VERSION}}
+      chart_name: charts/cf-review-env
+      chart_repo_url: ${{K8S_CHART_REPOSITORY}}
+      skip_cf_stable_helm_repo: true
+      kube_context: ${{K8S_CONTEXT_NAME}}
+      chart_version: 0.0.1+${{CF_SHORT_REVISION}}
```
Comment (victorpoluceno) [inline .codefresh/ci/pipeline.yml:28]: This is important so the CI pipeline doesn't deploy a the chart version.

File: .codefresh/cd/spec.yml [hunk 3]
```diff
@@ -43,6 +43,6 @@ spec:
   specTemplate:
     location: git
     repo: FindHotel/cf-review-env
-    path: ./.codefresh/build/pipeline.yml
-    revision: master
+    path: ./.codefresh/cd/pipeline.yml
+    revision: ch20439
```
Comment (rodrigo-findhotel) [inline .codefresh/cd/spec.yml:47]: Don't forget to change it to master

File: .codefresh/ci/spec.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  variables: []
+  contexts: []
+  externalResources: []
+  concurrency: 1
+  triggerConcurrency: 1
+  specTemplate:
+    location: git
+    repo: FindHotel/cf-review-env
+    path: ./.codefresh/ci/pipeline.yml
+    revision: ch20439
```
Comment (rodrigo-findhotel) [inline .codefresh/ci/spec.yml:49]: Don't forget to change it to master

**Pull request merged**